### DISCUSSION TITLE
Response model without validation

### DISCUSSION THREAD

**Discussion by victoraugustolls** [fastapi/fastapi#7604] [Questions] [answered]
### First check

* [X] I used the GitHub search to find a similar issue and didn't find it.
* [X] I searched the FastAPI documentation, with the integrated search.
* [X] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I add a response model but skip validation of it? I'm using ORJSONResponse always and I now prior to encoding that the data is correct.

**Comment by alexvanzyl** [comment]
@victoraugustolls Seem like there is currently a PR in the works #1434. It might be what you looking for.

**Comment by tiangolo** [comment, marked as answer]
If you set the `response_model` it will be used for documentation. By default, it will also be used for serialization, validation, filtering, etc. But if you return a response directly, then that overrides all the serialization, and the `response_model` would only be used for documentation in that case.

https://fastapi.tiangolo.com/advanced/response-directly/

Also, I assume you solved the issue, so thanks for coming back to close it! 🍰 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
